src/components/Chips.tsx

```diff
@@ -11,11 +12,14 @@ export interface ChipsProps<X> {
 
 /** Renders a list of `Chip`s, with wrapping & appropriate margin between each `Chip`. */
 export function Chips<X extends Only<ChipsXss, X>>(props: ChipsProps<X>) {
+  const { tableStyle } = usePresentationContext();
   const { values, xss = {} } = props;
+  // Do not allow Chips to wrap if within a `fixed` style table
+  const wrap = tableStyle !== "fixed";
```

Comment:
Hrm, I'm not a huge fan of components being coupled to "am I in a table or not", but we do have precedent of "use no labels / use compact" being set by the grid table.

Is there some attribute ... like "wrap" ... something ... that we could use/set here instead of just "I'm in a table"? 

I wonder if eventually our GridStyles will learn about presentation context settings, e.g. `const style1: GridStyle = ...` and `const style2: GridStyle = ...` would each have their own like `presentationContext: ...` sub-keys to set things like no labels/etc.

Reply:
Maybe even just `PresentationContext.wrap` if that's the most direct name for what this is controlling?